manifests-maistra/2.3.0/maistraoperator.v2.3.0.clusterserviceversion.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
       The Maistra Operator enables you to install, configure, and manage an instance of Maistra service mesh. Maistra is based on the open source Istio project.
 
     containerImage: quay.io/maistra/istio-ubi8-operator:2.3.0
-    createdAt: 2022-10-14T13:24:19EDT
+    createdAt: 2022-11-09T15:48:27EST
```

Comment:
Please revert this change. Make gen always updates the timestamp, but we usually revert it if it's the only change in the file.